Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php

```diff
@@ -77,7 +77,7 @@ class NodeTemplateConverter extends NodeConverter
      * @return mixed An object or \Neos\Error\Messages\Error if the input format is not supported or could not be converted for other reasons
      * @throws \Exception
      */
-    public function convertFrom($source, $targetType, array $subProperties = array(), PropertyMappingConfigurationInterface $configuration = null)
+    public function convertFrom($source, $targetType = null, array $subProperties = array(), PropertyMappingConfigurationInterface $configuration = null)
```

Comment:
Huh, but that isn't in sync with the interface's signature!? https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Classes/Property/TypeConverterInterface.php#L113

Reply:
It match the parent class ... but not the interface, look like a PHP bug ;)

Reply:
Ah, I see.. But that makes sense I guess: A subclass can make the argument optional but not the other way around

Reply:
Yes, sounds ok

Reply:
https://platform.sh/blog/watch-your-inheritance